woocommerce/woocommerce-ios
# Login: fix a crash after visiting site credentials screen and then auto-filling WP.com credentials

Closes: #7512 


- [x] ⚠️ Please review the WPAuthenticator PR https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/668, which contains most of the changes ⚠️ 
- [x] ⚠️ Make sure the WPAutheticator pod is using the production version in the Podfile ⚠️ 

### Testing instructions


- Launch the app, log out if needed, skip onboarding if needed
- Tap on “Enter Your Store Address”
- Enter a valid WP site address
- Tap “Sign in with store credentials”
- Go back to the email screen
- Tap on the email text field
- Tap on the password manager icon to auto-fill a WP.com email/password -> the app should not crash like how it does before this PR

---
